src/pages/dashboard/AdminMenu.tsx

```diff
@@ -34,11 +32,11 @@ const AdminMenu: FC = () => {
     <>
       {history.location.pathname !== '/new-publication' && (
         <StyledAdminMenu>
-          <MuiLink component={Link} to="/new-publication">
-            <StyledButton color="primary" variant="contained" data-testid="new-publication-button">
+          <StyledButtonWrapper>
+            <Button component={RouterLink} to="/new-publication" color="primary" data-testid="new-publication-button">
```

Comment:
Denne ser litt rar ut (ser ut som det er to buttons inni en):
![Skjermbilde 2020-02-20 kl  16 13 04](https://user-images.githubusercontent.com/6595982/74948041-04771b00-53fc-11ea-9f21-6331d7cabcd7.png)

Reply:
Ahh - StyledButtonWrapper må være en div. ikke en Button